CMP1903_A1_2324/Testing.cs

```diff
@@ -5,7 +5,52 @@ namespace Dice2
 {
 	public class Testing
 	{
-		
+		public Testing()
+		{
+		}
+
+		//Tests that the Die runs
+        internal void Run()
+        {
+			CreateDie();
+			TestRollDie();
```

Comment:
I think you also need to call the TestDice_WhenCallingRollMethod_ResultLessThanSeven() method here.